examples/process_script.py

```diff
@@ -23,34 +55,64 @@
     begin_line = datetime(2017, 11, 4, 0, 27)
     end_line = datetime(2017, 11, 4, 1, 45)
     gps_fields = ['mdy', 'hms', 'lat', 'long', 'ortho_ht', 'ell_ht', 'num_stats', 'pdop']
+    flight = gravity_file[4:11]
+    outdir = os.path.join(basedir, flight)
 elif campaign == 'OIB':
     print('OIB')
-    basedir = '/Users/dporter/Documents/Research/Projects/OIB-grav/data/P3_2017'
-    gravity_directory = 'gravity/dgs/raw/F2004'
-    gravity_file = 'OIB-P3_20170327_F2004_DGS_0938.dat'
-    trajectory_directory = 'pnt/dgs-ins/F2004/txt'
-    trajectory_file = 'OIB-P3_20170327_F2004_DGS-INS_RAPID_DGS.txt'
+    # # OIB P3 2017
+    # basedir = '/Users/dporter/Documents/Research/Projects/OIB-grav/data/P3_2017'
+#     gravity_directory = 'gravity/dgs/raw/F2004'
+#     gravity_file = 'OIB-P3_20170327_F2004_DGS_0938.dat'
+#     trajectory_directory = 'pnt/dgs-ins/F2004/txt'
+#     trajectory_file = 'OIB-P3_20170327_F2004_DGS-INS_RAPID_DGS.txt'
+    # # OIB GV 2019
+    basedir = '/Volumes/BASEQI/data_baseqi/OIB_GV_2019/F1014/Gravity/DgS/'
+    gravity_directory = basedir     # 'F1011'
+    # gravity_file = 'OIB-GV_F1008_20191102.dat'
+    gravity_file = 'OIB-GV_F1014_20191109.dat'
+    trajectory_directory = gravity_directory
+    # trajectory_file = 'OIB-GV_F1008_20191102_iMAR-INS_QC_DGS_TerraPOS.txt'
+    trajectory_file = 'OIB-GV_F1014_20191109_iMAR-INS_FINAL_DGS.txt'
+    # Moscow Univ
+    # begin_line = datetime(2017, 3, 27, 15, 45)
+    # end_line = datetime(2017, 3, 27, 16, 45)
+    #
+    begin_line = datetime(2019, 11, 9, 22, 0)
+    end_line = datetime(2019, 11, 10, 6, 0)
```

Comment:
I would consider exposing some of this stuff to generalize this script a bit.  If Caitlin is going to be using this script, it would be better if she weren't changing code.  Maybe a YAML or JSON file with some configuration info like this stuff.

Reply:
next push will have all these runtime configurations, like filenames, flight numbers, etc. in a YAML file